Userland/Libraries/LibJS/Runtime/Intl/PluralRules.cpp

```diff
@@ -154,14 +158,20 @@ ThrowCompletionOr<::Locale::PluralCategory> resolve_plural_range(VM& vm, PluralR
     // 7. Let yp be ! ResolvePlural(pluralRules, y).
     auto end_plurality = MUST_OR_THROW_OOM(resolve_plural(vm, plural_rules, end));
 
+    // 8. If xp.[[FormattedString]] is yp.[[FormattedString]], then
+    if (start_plurality.formatted_string == end_plurality.formatted_string) {
+        // a. Return xp.[[PluralCategory]].
+        return start_plurality.plural_category;
+    }
+
     // 8. Let locale be pluralRules.[[Locale]].
```

Comment:
Numbering is off here now

Reply:
:man_facepalming:  would've helped if I hit ctrl-s when I changed those numbers locally